ReactViewControl/WebPackDependenciesProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web.Script.Serialization;
+
+namespace ReactViewControl {
+    public class WebPackDependenciesProvider : IDependenciesProvider {
```

Comment:
this should be implemented out of this package... implemented at SS level and only included in debug config